### DISCUSSION TITLE
Is there a horisontal stacked bar chart?

### DISCUSSION THREAD

**Discussion by oscarthroedsson** [recharts/recharts#3808] [General]
Hey. I am searching for a single stacked bar chart for my web-page. Do we have anyone available?

**Comment by ckifer** [comment]
Hi

https://recharts.org/en-US/examples/StackedBarChart

Can you explain what you mean more than the example?

**Comment by oscarthroedsson** [comment]
![Skärmavbild 2023-10-03 kl  13 41 26](https://github.com/recharts/recharts/assets/74626974/62f42719-7a2a-49ed-8e2e-5358855bbff2)

Like this image. I just want it like that. Simple, clean =)

**Reply by ckifer** [reply]
Sure that should be possible. I'm not at a computer right now but maybe @nikolasrieble or @Yilun-Sun could help you out. The storybook also has more examples

**Reply by ckifer** [reply]
Here you are [![Edit stacked-bar-chart (forked)](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/stacked-bar-chart-forked-w473rm?fontsize=14&hidenavigation=1&theme=dark)

![image](https://github.com/recharts/recharts/assets/25180830/6937321f-6c9f-4577-ae8b-4d0427ba581f)

**Reply by ckifer** [reply]
From there you can hide the XAxis if wanted and remove the Grid
